src/main/webapp/app/course/manage/course-update.component.scss

```diff
@@ -23,6 +22,14 @@
         }
     }
 
+    .no-image.en {
+        background: #dddddd url('../../../content/images/image-placeholder-en.png') no-repeat center;
+    }
+
+    .no-image.de {
+        background: #dddddd url('../../../content/images/image-placeholder-de.png') no-repeat center;
```

Comment:
Can we use a <div> with a flexbox to render this with HTML instead of loading images?